src/main/java/Deadline.java

```diff
@@ -0,0 +1,46 @@
+public class Deadline extends Task{
+    private String time;
+    public Deadline(String command, boolean isDone, String time) {
+        super(command, isDone);
+        this.time = time;
+    }
+
+    public Deadline(String command, String time) {
+        super(command);
+        this.time = time;
+    }
+
+    public String getTime() {
+        return time;
+    }
+
+    public void setTime(String time) {
+        this.time = time;
+    }
+
+    @Override
+    public String getCommand() {
+        return super.getCommand();
+    }
+
+    @Override
+    public void setCommand(String command) {
+        super.setCommand(command);
+    }
+
+    @Override
+    public boolean isDone() {
+        return super.isDone();
+    }
+
+    @Override
+    public void setDone(boolean done) {
+        super.setDone(done);
+    }
```

Comment:
Inherited methods do not need to be redeclared if there are no changes to their functionality